origami-lib/test/server_textimageinput.py

```diff
@@ -0,0 +1,80 @@
+import sys
+from os import path
+# import parent directory into path so that origami can be imported
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+import origami
+import numpy as np
+import cv2      # for reading image files
+
+# running at 127.0.0.1
+# port 8888
+app = origami.register('gh:127.0.0.1:1:1:8888:0.0.0.0')
+
+# list of images to be received
+image_list = [
+    'example1.png',
+    'example2.png'
+]
+
+# list of text to be received
+text_list = [
+    'hello',
+    'world',
+    'Is this a sample question?'
+]
+
+@origami.crossdomain
+@app.listen()
+def test():
+    # test number: 
+    # 0 for text array, 
+    # 1 for image array using filepath, 
+    # 2 for image array using np array, 
+    # 3 for all
+    tnum = origami.request.form['test_number']
+
+    if tnum == '0' or tnum == '3':
+        print('hello')
+        # test getTextArray
+        all_text = origami.getTextArray()
+        print('Received %d text strings'%(len(all_text)))
+        print(all_text)
```

Comment:
You can send all_text as a response right?

Reply:
Yes. I could change that.

Reply:
yeah so do that. And do it for the file path and numpy array too :D

Reply:
Ok

Reply:
Hi, if I understand the code correctly, the file path is just a random string for the folder and the index for the filename right? If so, there would be no meaning returning it?